### PR TITLE
Adds github settings file for portable labels

### PR DESCRIPTION
**Pull request by danimtb**
Has to be merged to develop branch for changes to take effect

After merge we need to activate the app here: https://probot.github.io/apps/settings/

### CODE REVIEW

**Comment by danimtb** [comment]
CI was cancel as it is not affected by this PR

**Comment by jgsogo** [comment]
Do you want to keep this file almost-equal for `conan-io/conan` and `conan-community/community` too, or it is just intended for this repo?

If the answer is the first one, then it is ok, otherwise I would add the `has_projects=False` too (I don't know if `has_downloads=False` is needed too).

File: .github/settings.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# https://github.com/apps/settings
+
+repository:
```
Comment (jgsogo) [inline .github/settings.yml]: What about the name?

```suggestion
repository:
  name: docs
```

Comment (danimtb) [reply .github/settings.yml]: I think I am going to keep it as generic as possible to try to have the same settings for all repositories

Comment (jgsogo) [reply .github/settings.yml]: and what about the `description` and `homepage`?

Comment (danimtb) [reply .github/settings.yml]: Yes, I am going to remove them too

**Comment by jgsogo** [comment]
What about this (https://github.com/probot/probot-config) to share configuration between repos? Will it work? Having a `conan-io/conan: .github/common-settings.yml` and add a reference (and specialize) to it from the `.github/settings.yml` file of every repo

**Comment by danimtb** [comment]
I think we could at this in the future if this kind of settings start to grow. I don't see this labels changing very much in a short term and configs are easy enough to be copy-pasted 😄 

However, if you feel so we can start putting this common labels in conan-io/conan repo and the reuse them in here... I just wanted to start with a smaller repo like this one

**Comment by danimtb** [comment]
I am the one doing it! 🐮 🤠

**Comment by danimtb** [comment]
Done! Bu labels were not update 🤦‍♂️ Maybe we need to push a new change with the app now active... 🤔

**Comment by jgsogo** [comment]
Sure, an empty commit to `develop` should be enough

**Pull request merged**